tests/internal/discoverytest/discoverytest.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !windows
```

Comment:
Why can't we run it on Windows?

Reply:
Nevermind, I see why

Reply:
`syscall.Stat_t` is not available on Windows